src/main/java/com/nulabinc/zxcvbn/Scoring.java

```diff
@@ -12,6 +12,8 @@
 
     public static final int MIN_GUESSES_BEFORE_GROWING_SEQUENCE = 10000;
 
+    public static final long JS_NUMBER_MAX = 9007199254740992L;
```

Comment:
@InkerBot Sorry for the late reply. JS_NUMBER_MAX is JavaScript's Number.MAX_SAFE_INTEGER, right? The following documentation seems to indicate that `9007199254740991` (9007199254740991L) is the correct value.

> The Number.MAX_SAFE_INTEGER constant represents the maximum safe integer in JavaScript (2^53 - 1).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER